app/admin/dashboard.rb

```diff
@@ -483,7 +483,7 @@ def export_metrics
             column :total_emails
           end
 
-          h4("Email Counts by Practice", title: "Number of times a practice was emailed via the practice page for each practice", class: "dm-tooltip")
+          h4("Email Counts by Practice", title: "Number of times an innovation was emailed via the practice page for each practice", class: "dm-tooltip")
```

Comment:
for example here it should be "Email Counts by Innovation"

Reply:
Got it.. I'll look in admin panel for more cases where users could see the term practice.